src/modm/board/nucleo_g071rb/board.hpp

```diff
@@ -70,7 +70,7 @@ struct SystemClock
 	static constexpr uint32_t Usart3	= Apb;
 	static constexpr uint32_t Usart2	= Apb;
 	static constexpr uint32_t Spi2		= Apb;
-	static constexpr uint32_t Iwdg		= Apb;
+	static constexpr uint32_t Iwdg		= Rcc::LsiFrequency;
```

Comment:
You wanted to make the same change to the G070 Nucleo as well. Alternatively you could merge the headers.